### PR TITLE
Add hyperlinks to some notes

### LINKED ISSUE #1462: add a hyperlink to the comment
## Question

I think we should add a hyperlink to the comment to quickly find the code source.

such: _https://github.com/apache/dubbo/blob/master/dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java_ in ExtensionLoader class

### PR DESCRIPTION
**Pull request by THyyy**
Fixes #1462 add a hyperlink to the comment for whole project.

Make sure that:
- [x] You have read the [contribution guidelines](https://dromara.org/projects/soul/contributor/).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Review by tuohai666 — APPROVED** [review]

**Comment by tuohai666** [comment]
@THyyy Please fix the ci.

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/exception/ExceptionHandlers.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 /**
  * ControllerMethodResolver.
- * https://dzone.com/articles/global-exception-handling-with-controlleradvice.
+ * @see <a href="https://dzone.com/articles/global-exception-handling-with-controlleradvice">global-exception-handling-with-controlleradvice</a>.
```
Comment (tuohai666) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/exception/ExceptionHandlers.java:33]: Remove '.' at the end of this line.

**Review by tuohai666 — CHANGES_REQUESTED** [review]

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/exception/ExceptionHandlers.java (hunk 1 above)
Comment (THyyy) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/exception/ExceptionHandlers.java]: ok，I fixed it and push the code to git repository

**Comment by THyyy** [comment]
maven-checkstyle-plugin‘s 3.1.0 version makes a new question.

**Comment by THyyy** [comment]
maven-checkstyle-plugin‘s 3.1.0 version makes a new question.

**Review by tuohai666 — DISMISSED** [review]

**Comment by tuohai666** [comment]
@THyyy Please fix the new ci failure.

**Comment by tuohai666** [comment]
I suggest you run the` mvn clean install` instead of `mvn clean install -Dmaven.javadoc.skip=true` in local to have a test first.

**Review by dengliming — APPROVED** [review]

**Pull request merged**